OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java

```diff
@@ -38,6 +37,12 @@ public void onCreate(Bundle savedInstanceState) {
     setTitle(getName());
 
     selectionMap = new LinkedHashMap<String, PlayerSelectionOption>();
+    selectionMap.put("HLS VOD", new PlayerSelectionOption("A0NW1vYTE6hCIME00w2On3_iHKOCE6Z7", "x2aDkyOt2q3WtOCp-krSyDffASzL", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("eHLS VOD", new PlayerSelectionOption("hneHVnMzE6sEsCT9DE1v_yWi0o-9aCTw", "x2aDkyOt2q3WtOCp-krSyDffASzL", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("DASH VOD", new PlayerSelectionOption("g3aDUzODE6EWDhHR_FDxuCSPRTBCE1Gq", "hjZ2gyOn8Y0z7pvLjDdvaibqJ-dR", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("DRM DASH", new PlayerSelectionOption("Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm", "FoeG863GnBL4IhhlFC1Q2jqbkH9m", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
```

Comment:
The DRM one is not playing yet. You need to add this one in the ContentProtectionSampleApp anyway.

Reply:
Moved the DRM to the ContentProtectionSampleApp. Thanks.